src/Backend/Modules/Groups/Entity/Group.php

```diff
@@ -0,0 +1,215 @@
+<?php
+
+namespace Backend\Modules\Groups\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+use Doctrine\Common\Collections\ArrayCollection;
+
+/**
+ * This is the Group Entity
+ *
+ * @author Mathias Dewelde <mathias@dewelde.be>
+ *
+ * @ORM\Entity
+ * @ORM\Table(name="Groups")
```

Comment:
You should use the old database table name here. So nothing breaks in other modules.
After the full Doctrine switch is done, we can remove this line in all Entities.
```php
 * @ORM\Table(name="groups")
```